README.md

```diff
@@ -12,9 +12,9 @@ Run the following commands on an all-snap image:
 
     $ sudo snap install --beta --devmode classic
     $ sudo classic
-    (classic)ubuntu@localhost:~$ sudo apt update
```

Comment:
Maybe `apt install hello` then or something? Just some example to show that its a "regular" system :)